doc/sphinx/practical-tools/coqide.rst

```diff
@@ -5,130 +5,183 @@
 Coq Integrated Development Environment
 ========================================
 
-The Coq Integrated Development Environment is a graphical tool, to be
-used as a user-friendly replacement to `coqtop`. Its main purpose is to
-allow the user to navigate forward and backward into a Coq
-file, executing corresponding commands or undoing them respectively.
-
-CoqIDE is run by typing the command `coqide` on the command line.
-Without argument, the main screen is displayed with an “unnamed
-buffer”, and with a filename as argument, another buffer displaying
-the contents of that file. Additionally, `coqide` accepts the same
-options as `coqtop`, given in :ref:`thecoqcommands`, the ones having obviously
-no meaning for CoqIDE being ignored.
+.. todo PR: how to say that a number of things are broken?  Maybe list them
+   somewhere--doesn't have to be super detailed
+
+The Coq Integrated Development Environment (CoqIDE) is a user-friendly GUI
+for Coq. Its main purpose is to
+allow users to edit scripts and step forward and backward through Coq scripts,
+respectively executing and undoing commands and tactics.
```

Comment:
This paragraph is hard to read IMO.  
```suggestion
The Coq Integrated Development Environment (CoqIDE) is a user-friendly GUI
for Coq. Its main purpose is to
allow users to edit scripts and step through Coq scripts, executing and undoing commands and tactics.
```

The respectively part encourages me to reread, but who knows, maybe I can't read.

Reply:
Also please rehard wrap.

Reply:
Rewrapping is a manual process.  My GUI doesn't have a command to do that.  Also it doesn't matter, does it?  We expect users to read the HTML.  But I did rewrap this couple of lines.

Reply:
@jfehrle My GUI allows me to do some rewrapping, it can help when reading the rst files directly, which is sometimes done. It also helps people when they edit it later. Not a big deal if you cannot do it right now however.

Reply:
It's OK to rewrap the changed parts, but en masse rewrapping would obscure the git history.  I expect this is not news to you.